### PR TITLE
Updating changelog for 10.9.0

### PR DESCRIPTION
**Pull request by ryanml**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [10.9.0]
 ### Uncategorized
-- Version v10.9.0
-- EIP-1559 V2: Use default gas preferences of user for all transactions ([#13238](https://github.com/MetaMask/metamask-extension/pull/13238))
 - Handle missing eth_accounts permission history in getLastConnectedInfo ([#13257](https://github.com/MetaMask/metamask-extension/pull/13257))
```
Comment (danjm) [inline CHANGELOG.md:11]: I don't think https://github.com/MetaMask/metamask-extension/pull/13257 is end user facing

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Alert users when the network is busy ([#12268](https://github.com/MetaMask/metamask-extension/pull/12268))
-- Clear Sass deprecations ([#13227](https://github.com/MetaMask/metamask-extension/pull/13227))
-- Add JSDoc ESLint rules ([#12112](https://github.com/MetaMask/metamask-extension/pull/12112))
-- [Hotfix] track anonymous events appropriately ([#13237](https://github.com/MetaMask/metamask-extension/pull/13237))
-- Remove dupe Prettier config from ESLint config ([#13234](https://github.com/MetaMask/metamask-extension/pull/13234))
-- EIP-1559 V2 UI fixes ([#13222](https://github.com/MetaMask/metamask-extension/pull/13222))
-- Dark Mode Part 1: Switch to Using CSS Variables ([#13147](https://github.com/MetaMask/metamask-extension/pull/13147))
-- using 1559 V2 for cancel speed up flows ([#13019](https://github.com/MetaMask/metamask-extension/pull/13019))
-- Update Flask support links and home footer ([#13226](https://github.com/MetaMask/metamask-extension/pull/13226))
 - fix issue where transaction items without a TRANSACTION_CONFIRMED_EVENT have unhandled error when clicking into activity log ([#13221](https://github.com/MetaMask/metamask-extension/pull/13221))
```
Comment (danjm) [inline CHANGELOG.md:13]: We can probably modify this description

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Sync `develop` with `master` ([#13115](https://github.com/MetaMask/metamask-extension/pull/13115))
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- Switch Flask horizontal logos ([#13113](https://github.com/MetaMask/metamask-extension/pull/13113))
-- Update `@babel/runtime` patch to fix lockdown error ([#13109](https://github.com/MetaMask/metamask-extension/pull/13109))
 - Fix account name duplicates ([#12867](https://github.com/MetaMask/metamask-extension/pull/12867))
 - Choose accounts refactor ([#13039](https://github.com/MetaMask/metamask-extension/pull/13039))
 - Fix permissions-connect-footer "learn more" link ([#13092](https://github.com/MetaMask/metamask-extension/pull/13092))
-- Master <> Develop Sync (post-v10.8.0) ([#13088](https://github.com/MetaMask/metamask-extension/pull/13088))
-- Feat/collectibles the return ([#12970](https://github.com/MetaMask/metamask-extension/pull/12970))
 - Subject metadata cleanup ([#13090](https://github.com/MetaMask/metamask-extension/pull/13090))
```
Comment (danjm) [inline CHANGELOG.md:25]: This one can be removed

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Fix merge conflict typo
-- Merge branch 'develop' into master-sync
-- Bump just-safe-set from 2.1.0 to 2.2.3 ([#13049](https://github.com/MetaMask/metamask-extension/pull/13049))
 - Fix typo in German translation ([#13040](https://github.com/MetaMask/metamask-extension/pull/13040))
-- Using EIP-1559 V2 for swaps ([#12966](https://github.com/MetaMask/metamask-extension/pull/12966))
 - Make restore vault a form so an user can submit via keyboard ([#12989](https://github.com/MetaMask/metamask-extension/pull/12989))
 - Remove legacy node parent detection ([#12814](https://github.com/MetaMask/metamask-extension/pull/12814))
```
Comment (danjm) [inline CHANGELOG.md:28]: We can remove this one

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Add stories for Home notification component ([#13035](https://github.com/MetaMask/metamask-extension/pull/13035))
-- Update Redux DevTools README instructions ([#13038](https://github.com/MetaMask/metamask-extension/pull/13038))
-- Jestify app/scripts/controller/network/**/*.test.js ([#12985](https://github.com/MetaMask/metamask-extension/pull/12985))
 - Fix order of account list ([#12999](https://github.com/MetaMask/metamask-extension/pull/12999))
-- Changes in gas loading animation in EIP-1559 V2 ([#13016](https://github.com/MetaMask/metamask-extension/pull/13016))
 - Add crowdin configuration and github action ([#12552](https://github.com/MetaMask/metamask-extension/pull/12552))
```
Comment (danjm) [inline CHANGELOG.md:30]: We can remove this one

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Wire up network stability slider in new gas modal ([#13029](https://github.com/MetaMask/metamask-extension/pull/13029))
```
Comment (danjm) [inline CHANGELOG.md:31]: This is still behind a feature flag

File: CHANGELOG.md [hunk 7]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Update ESLint config to v8 ([#12886](https://github.com/MetaMask/metamask-extension/pull/12886))
-- Update `electron` from 11.4.12 to 11.5.0 ([#13037](https://github.com/MetaMask/metamask-extension/pull/13037))
-- Add `@metamask/rpc-methods` package ([#13027](https://github.com/MetaMask/metamask-extension/pull/13027))
 - display hex data on send screen fixed ([#12709](https://github.com/MetaMask/metamask-extension/pull/12709))
 - Support EIP-1559 on Trezor Model One ([#13023](https://github.com/MetaMask/metamask-extension/pull/13023))
```
Comment (danjm) [inline CHANGELOG.md:33]: This was part of a past release

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Add subject types ([#13026](https://github.com/MetaMask/metamask-extension/pull/13026))
```
Comment (danjm) [inline CHANGELOG.md:34]: Not user facing

File: CHANGELOG.md [hunk 9]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Update LavaMoat policies ([#12992](https://github.com/MetaMask/metamask-extension/pull/12992))
-- Fixing EIP-1559 V2 advance gas fee inputs validations ([#13013](https://github.com/MetaMask/metamask-extension/pull/13013))
-- Remove `nodeify` ([#13003](https://github.com/MetaMask/metamask-extension/pull/13003))
 - "eth_signTypedData" presents fields that do not appear in 'types' filed ([#12905](https://github.com/MetaMask/metamask-extension/pull/12905))
 - toggle-button ([#12817](https://github.com/MetaMask/metamask-extension/pull/12817))
```
Comment (danjm) [inline CHANGELOG.md:36]: Not user facing ( a storybook change)

File: CHANGELOG.md [hunk 10]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Fix error-message component to match new Storybook format ([#12815](https://github.com/MetaMask/metamask-extension/pull/12815))
-- Fix dropdown component for new Storybook format ([#12816](https://github.com/MetaMask/metamask-extension/pull/12816))
-- Fix signature-request for new Storybook format ([#12755](https://github.com/MetaMask/metamask-extension/pull/12755))
-- Fix form-field for new Storybook format ([#12756](https://github.com/MetaMask/metamask-extension/pull/12756))
 - Identicon size fix ([#13014](https://github.com/MetaMask/metamask-extension/pull/13014))
 - Remove unnecessary action text property from whats new notification 9 ([#13021](https://github.com/MetaMask/metamask-extension/pull/13021))
 - Dispose the trezor connect iframe upon lock ([#13018](https://github.com/MetaMask/metamask-extension/pull/13018))
```
Comment (danjm) [inline CHANGELOG.md:39]: Part of a past release

File: CHANGELOG.md [hunk 11]
```diff
@@ -108,28 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Consolidate permission exports ([#13012](https://github.com/MetaMask/metamask-extension/pull/13012))
```
Comment (danjm) [inline CHANGELOG.md:43]: Not user facing

File: CHANGELOG.md [hunk 12]
```diff
@@ -108,28 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Fix browser specific manifest generation ([#13007](https://github.com/MetaMask/metamask-extension/pull/13007))
```
Comment (danjm) [inline CHANGELOG.md:44]: Not user facing

File: CHANGELOG.md [hunk 13]
```diff
@@ -108,28 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Fix metamask-translation component for new Storybook format ([#12892](https://github.com/MetaMask/metamask-extension/pull/12892))
```
Comment (danjm) [inline CHANGELOG.md:45]: Not user facing

File: CHANGELOG.md [hunk 14]
```diff
@@ -108,28 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Sync develop with master ([#13010](https://github.com/MetaMask/metamask-extension/pull/13010))
-- Merge remote-tracking branch 'origin/develop' into master-sync
 - Tx decoding accreditation ([#12918](https://github.com/MetaMask/metamask-extension/pull/12918))
```
Comment (danjm) [inline CHANGELOG.md:46]: Part of a past release

File: CHANGELOG.md [hunk 15]
```diff
@@ -108,28 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Adding featureflags to .metamaskrc.dist file ([#12962](https://github.com/MetaMask/metamask-extension/pull/12962))
 - Update tx-insights url ([#12997](https://github.com/MetaMask/metamask-extension/pull/12997))
```
Comment (danjm) [inline CHANGELOG.md:47]: Part of a past release

File: CHANGELOG.md [hunk 16]
```diff
@@ -108,28 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Updating storybook documentation guidelines ([#12957](https://github.com/MetaMask/metamask-extension/pull/12957))
 - Fix confirmed transaction time  ([#12633](https://github.com/MetaMask/metamask-extension/pull/12633))
-- Fix Actionable message for new Storybook format ([#12874](https://github.com/MetaMask/metamask-extension/pull/12874))
-- Fix icon component for new Storybook format ([#12887](https://github.com/MetaMask/metamask-extension/pull/12887))
-- Fix identicon component for new Storybook format ([#12888](https://github.com/MetaMask/metamask-extension/pull/12888))
-- text-field ([#12889](https://github.com/MetaMask/metamask-extension/pull/12889))
-- Fix account-list-item component for new Storybook format ([#12893](https://github.com/MetaMask/metamask-extension/pull/12893))
-- Fix dropdown-input-pair component to for new Storybook format ([#12895](https://github.com/MetaMask/metamask-extension/pull/12895))
-- Fix popover component to for new Storybook format ([#12896](https://github.com/MetaMask/metamask-extension/pull/12896))
```
Comment (danjm) [inline CHANGELOG.md:124]: A lot of the ones below are part of a past release

File: CHANGELOG.md [hunk 17]
```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Update the Sentry setup log with the build type ([#13157](https://github.com/MetaMask/metamask-extension/pull/13157))
-- maintain nft image aspect ratios and center images in collection view ([#13136](https://github.com/MetaMask/metamask-extension/pull/13136))
-- Remove duplicate selector `getPermissionSubjectsMetadata` ([#13144](https://github.com/MetaMask/metamask-extension/pull/13144))
-- Add Chrome store policy compliant names/descriptions ([#13142](https://github.com/MetaMask/metamask-extension/pull/13142))
-- Sync `master` with `develop` ([#13141](https://github.com/MetaMask/metamask-extension/pull/13141))
-- Merge remote-tracking branch 'origin/develop' into sync-master
 - Add more missing `appName` messages ([#13140](https://github.com/MetaMask/metamask-extension/pull/13140))
-- EIP-1559 V2: Adding default settings to advanced gas modal ([#12911](https://github.com/MetaMask/metamask-extension/pull/12911))
-- EIP-1559 V2: Advanced gas fee modal - base fee and priority fee trends ([#13025](https://github.com/MetaMask/metamask-extension/pull/13025))
 - Wire up priority fee range in new gas modal ([#13044](https://github.com/MetaMask/metamask-extension/pull/13044))
```
Comment (danjm) [inline CHANGELOG.md:21]: Behind a feature flag

**Comment by tmashuang** [comment]
I have an alternative PR for the changelog stuffs, lmk what yall think, #13364

**Comment by danjm** [comment]
@tmashuang's alternative looks good

**Pull request closed without merging**